pac/tasks/update-deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-deployment
+spec:
+  description: Task to update deployment with newly built image in gitops repository.
+  params:
+    - name: gitops-repo-url
+      type: string
+      description: URL of gitops repository to update with the newly built image.
+    - name: image
+      type: string
+      description: Reference of the newly built image to use.
+    - name: gitops-auth-secret-name
+      type: string
+      default: gitops-auth-secret
+      description: |
+        Secret of basic-auth type containing credentials to commit into gitops repository.
+  volumes:
+    - name: gitops-auth-secret
+      secret:
+        secretName: $(params.gitops-auth-secret-name)
+        optional: true
+  steps:
+    - name: patch-gitops
+      image: quay.io/konflux-ci/appstudio-utils:ab6b0b8e40e440158e7288c73aff1cf83a2cc8a9@sha256:24179f0efd06c65d16868c2d7eb82573cce8e43533de6cea14fec3b7446e0b14
+      volumeMounts:
+        - name: gitops-auth-secret
+          mountPath: /gitops-auth-secret
+      env:
+        - name: PARAM_GITOPS_REPO_URL
+          value: $(params.gitops-repo-url)
+        - name: PARAM_IMAGE
+          value: $(params.image)
+      script: |
+        if test -f /gitops-auth-secret/password ; then
```

Comment:
so this is the old update deployment?

Reply:
we should probably switch like the RHTAP task but we can do that after merging and installer has been delivered

Reply:
Yeah I believe this way matches the current setup of the installer